woocommerce/wc-admin
# Pills and tooltips in product step of OBW

Partially addresses #4577

This replaces the existing price text with a new pill component, using per-month amounts. It also adds a tooltip to each pill.

### Accessibility


### Screenshots
![image](https://user-images.githubusercontent.com/224531/86303724-0d041b80-bc50-11ea-8707-66b6d31082c9.png)

Smaller viewport:
![image](https://user-images.githubusercontent.com/224531/86508015-e09ef980-be1f-11ea-91fd-fec37ab6c04f.png)

### Detailed test instructions:

- Start the OBW
- Step through to the product selection task
- Check that the layout matches the [design mockup](https://www.figma.com/file/JH9XMFUCOjfXdr3N09AHRD/On-boarding-iterations-June-'20?node-id=260%3A1491)
